lib/airbrake-ruby/filter_chain.rb

```diff
@@ -13,7 +13,8 @@ class FilterChain
       notice[:errors].each do |error|
         Gem.path.each do |gem_path|
           error[:backtrace].each do |frame|
-            frame[:file].sub!(/\A#{gem_path}/, '[GEM_ROOT]'.freeze)
+            next unless (file = frame[:file])
```

Comment:
Would be nice to have a comment here repeating what you said in PR description "when airbrake-ruby tries to send an exception with a backtrace, which couldn't be parsed. In that case we put the whole frame into function, leaving line and file nil."